pkg/controller/migmigration/backup.go

```diff
@@ -174,6 +174,12 @@ func (t Task) hasBackupCompleted(backup *velero.Backup) (bool, []string) {
 				backup.Name))
 	case velero.BackupPhaseFailedValidation:
 		reasons = backup.Status.ValidationErrors
+		reasons = append(
+			reasons,
+			fmt.Sprintf(
+				"Backup: %s/%s validation failed.",
+				backup.Namespace,
+				backup.Name))
```

Comment:
While troubleshooting, I noticed the backup information was not being added to `Errors` for validation failures.  This really helps with investigation.